src/libs/actions/connections/index.ts

```diff
@@ -46,7 +46,7 @@ function updatePolicyConnectionConfig<TConnectionName extends ConnectionName, TS
     policyID: string,
     connectionName: TConnectionName,
     settingName: TSettingName,
-    settingValue: Connections[TConnectionName]['config'][TSettingName] | null,
+    settingValue?: Connections[TConnectionName]['config'][TSettingName],
```

Comment:
This should be required. If we send an undefined value nothing will happen, i.e. Onyx will not set the value to `undefined`, it will do nothing as merging with undefined values is a NOP.

Reply:
@s77rt Still not sure - what should be send in that case? I see that we have optional fields to send in Onyx. Not sure why we could not do it here?